frontend/src/app/modules/invite-user-modal/principal/principal-search.component.ts

```diff
@@ -49,12 +50,12 @@ export class PrincipalSearchComponent extends UntilDestroyedMixin implements OnI
     this.input$,
     this.currentUserService.hasCapabilities$('users/create'),
   ).pipe(
-    map(([elements, input, canCreateUsers]) =>
-      canCreateUsers
+    map(([elements, input, canCreateUsers]) => {
+      return canCreateUsers
       && this.type === PrincipalType.User
       && input?.includes('@')
```

Comment:
Ensure returned type is `boolean`
```suggestion
      && !!input?.includes('@')
```